CHANGELOG.md

```diff
@@ -1,3 +1,23 @@
+# dbt_reddit_ads_source v0.3.0
+[PR #7](https://github.com/fivetran/dbt_reddit_ads_source/pull/7) includes the following **BREAKING CHANGE** updates:
+
+## Feature: Conversion Metrics
+- Introduces 4 new staging models to bring in conversion metrics across different dimensions:
+  - `stg_reddit_ads__account_conversions_report`
+  - `stg_reddit_ads__ad_group_conversions_report`
+  - `stg_reddit_ads__ad_conversions_report`
+  - `stg_reddit_ads__campaign_conversions_report`
```

Comment:
We should probably make a note to ensure customers have these tables syncing in their connector. Although it won't cause any failures (due to the union_data empty table) it will result in no results. So we should call this out here.